package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "fs-extra": "^0.16.3",
     "q": "~1.0",
-    "svgobjectmodelgenerator": "git+https://github.com/adobe-research/svgObjectModelGenerator.git#release-0.6",
+    "svgobjectmodelgenerator": "git+https://github.com/adobe-photoshop/svgObjectModelGenerator.git#release-0.6",
```

Comment:
The release hash is the same, that seems wrong.

Reply:
in this case it refers to a branch.  So it works.  I don't think it is "wrong".  I could change it to a specific tag name if you'd prefer @baaygun

Reply:
Oh, duh. Got thrown off by the `release`. Ok commit seems right in npm-shrinkwrap